spaceship.zsh

```diff
@@ -928,9 +928,9 @@ spaceship_ember() {
   # Show EMBER status only for folders w/ ember-cli-build.js files
   [[ -f ember-cli-build.js ]] || return
```

Comment:
@thebluejay you need the check if the ember-cli package.json exists or you get the following:

![screen shot 2017-06-27 at 4 39 10 pm](https://user-images.githubusercontent.com/575938/27614514-2c6daeee-5b57-11e7-92a9-e807f3dc9295.png)